packages/providers/mgt-msal2-provider/src/Msal2Provider.ts

```diff
@@ -733,8 +733,8 @@ export class Msal2Provider extends IProvider {
    * @return {*}  {Promise<string>}
    * @memberof Msal2Provider
    */
-  public async getAccessToken(options?: AuthenticationProviderOptions): Promise<string> {
-    const scopes = options ? options.scopes || this.scopes : this.scopes;
+  public async getAccessToken(opts?: AuthenticationProviderOptions): Promise<string> {
+    const scopes = opts && opts?.scopes?.length ? opts.scopes : this.scopes;
```

Comment:
We're already checking if `opts` is defined with `opts?`
```suggestion
    const scopes = opts?.scopes?.length ? opts.scopes : this.scopes;
```